Acceleratio.XCellKit/SpredsheetChart.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+
+namespace Acceleratio.XCellKit
+{
+    public class SpredsheetChart
+    {
+        public ChartTypeEnum ChartType { get; private set; }
+        public List<ChartModel> ChartData { get; private set; }
+        public List<GanttData> GanttData { get; private set; }
+        public ChartSettings Settings { get; set; }
+
+        protected SpredsheetChart(SpredsheetChart export)
+        {
+            this.ChartType = export.ChartType;
+            this.ChartData = export.ChartData;
+            this.GanttData = export.GanttData;
+            this.Settings = export.Settings;
+        }
+
+        public SpredsheetChart(ChartTypeEnum chartType, List<ChartModel> chartData, ChartSettings settings)
+        {
+            this.ChartType = chartType != ChartTypeEnum.Gantt ? chartType : ChartTypeEnum.Bar;
+            this.ChartData = chartData.Distinct().ToList();
+            this.Settings = settings;
+        }
+
+        public SpredsheetChart(List<GanttData> ganttData, ChartSettings settings)
```

Comment:
Maybe add comment on this constructor so the user knows the Gantt Chart is created. Maybe better approach would be to create additional class GanttSpreadsheetChart that inherits from SpredsheetChart? This usage is a bit misleading.